packages/ddg2dnr/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es2021",
+    "moduleResolution": "node",
+    "alwaysStrict": true,
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "declaration": true,
+    "strict": true,
+    "noImplicitAny": false,
+    "strictNullChecks": true,
+    "maxNodeModuleJsDepth": 0
+  },
+  "include": [
+      "*.js",
+      "**/*.js",
+      "node_modules/@duckduckgo/privacy-grade/src/classes/trackers.js"
+  ],
+  "types": [
+      "chrome"
+  ]
```

Comment:
This is incorrect (should be inside compilerOptions) - but actually it's not needed at all - TS will auto-include npm dependencies that are in `node_modules`
```suggestion
```

Reply:
@sammacbeth Sorry if this wasn't part of the PR, but we might as well fix now :)